content/ja/docs/concepts/security/secrets-good-practices.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Kubernetes Secretの適切な使用方法
+description: >
+  クラスタ管理者とアプリケーション開発者向けの適切なSecret管理の原則と実践方法。
+content_type: concept
+weight: 70
+---
+
+<!-- 概要 -->
+
+{{<glossary_definition prepend="Kubernetesでは、Secretは次のようなオブジェクトです。"
+term_id="secret" length="all">}}
+
+以下の適切な使用方法は、クラスタ管理者とアプリケーション開発者の両方を対象としています。
+これらのガイドラインに従って、Secretオブジェクト内の機密情報のセキュリティを向上させ、
+Secretの効果的な管理を行ってください。
+
+<!-- 本文 -->
+
+## クラスタ管理者
+
+このセクションでは、クラスタ管理者がクラスタ内の機密情報のセキュリティを強化するために使用できる適切な方法を提供します。
+
+### データの安全な暗号化を構成する
+
+デフォルトでは、Secretオブジェクトは{{<glossary_tooltip
+term_id="etcd" text="etcd">}}内で暗号化されていない状態で保存されます。
+`etcd`内のSecretデータを暗号化するように構成する必要があります。
+手順については、[データの安全な暗号化](/docs/tasks/administer-cluster/encrypt-data/)を参照してください。
+
+### Secretへの最小特権アクセスを構成する {#least-privilege-secrets}
+
+Kubernetesの{{<glossary_tooltip term_id="rbac" text="ロールベースアクセス制御">}} [(RBAC)](/docs/reference/access-authn-authz/rbac/)などのアクセス制御メカニズムを計画する際、`Secret`オブジェクトへのアクセスに関する以下のガイドラインを考慮してください。
+また、[RBACの適切な使用方法](/docs/concepts/security/rbac-good-practices)の他のガイドラインにも従ってください。
+
+- **コンポーネント**: `watch`または`list`アクセスを、特権のあるシステムレベルのコンポーネントに制限してください。コンポーネントの通常の動作が必要とする場合にのみ、Secretへの`get`アクセスを許可してください。
+- **ユーザー**: Secretsへの`get`、`watch`、`list`アクセスを制限してください。`etcd`へのアクセスはクラスタ管理者にのみ許可し、読み取り専用アクセスも許可してください。特定の注釈を持つSecretへのアクセスを制限するなど、より複雑なアクセス制御については、サードパーティの認証メカニズムを検討してください。
+
+{{< caution >}}
```

Comment:
{{<caution>}}
{{</caution>}}
I think should be like this

Reply:
According to the [Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/#caution), the following format is correct.
```
{{< caution >}}
{{< /caution >}}
```
On the other hand, the last sentnce in this PR uses `{{<caution>}} ... {{</caution>}}`.
This is the same notation as the original page, and works in this format as well.
https://github.com/kubernetes/website/blob/4d8feb6765a52362db09ef8d5dce41f45bc5fe9b/content/en/docs/concepts/security/secrets-good-practices.md?plain=1#L113-L116
If fix it to the correct notation, it is better to update the original page at first.

Reply:
My comment means the correct format is **with spaces**.
And for multiple language modifications, need to create different PRs.